packages/fast-usdc/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "fast-usdc",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Create an Agoric Javascript smart contract application",
```

Comment:
This seems generic, is there a better description we can give?

Reply:
oops, good catch. that was copied from create-dapp. I'll fix it in a future PR or this one if I need to rev CI